pages/test_analytics/python_collectors.md.erb

```diff
@@ -1,39 +1,54 @@
-# Configuring Python with Test Analytics
-
-To use Test Analytics with your Python projects, generate JUnit XML files with [pytest](https://docs.pytest.org), and then [upload the JUnit XML files](/docs/test-analytics/importing-junit-xml) to Test Analytics.
-
-{:notoc}
-
-1. Install [pytest](https://docs.pytest.org/en/6.2.x/getting-started.html):
-
-    ```sh
-    pip install -U pytest
+# Python collectors
+
+Test Analytics collectors for Python are provided by the [`buildkite-test-collector`](https://pypi.org/project/buildkite-test-collector/) package.
+You can also upload test results by importing [JSON](/docs/test-analytics/importing-json) or [JUnit XML](/docs/test-analytics/importing-junit-xml).
+
+{:toc}
+
+## pytest collector
+
+pytest is a testing framework for Python.
+If you're already using pytest, then you can install `buildkite-test-collector` to collect test results into your Test Analytics dashboard.
+
+Before you start, make sure pytest runs with access to [CI environment variables](/docs/test-analytics/ci-environments).
```

Comment:
It depends. 😒

This wording comes from some of the other collector docs, so I didn't attempt to fix this problem here.

The case of a plain `command` step doesn't require anything extra, since it runs in the same shell as the agent. This line is most significant if the tests run inside a container. By default, agents won't export environment variables to containers, so the actual how-to coverage for this would be multifacted depending on the containerization tooling (e.g., docker versus docker-compose versus podman versus etc.) and the CI system (e.g., Buildkite, CircleCI, GitHub Actions, etc.).